tests/state_test.go

```diff
@@ -29,10 +29,14 @@ import (
 	"testing"
 
 	"github.com/kaiachain/kaia/blockchain/vm"
+	"github.com/kaiachain/kaia/common"
 	"github.com/kaiachain/kaia/params"
 )
 
 func TestState(t *testing.T) {
+	common.RelaxPrecompileRangeForTest(false)
+	enableTestExecutionSpecState(false)
```

Comment:
To not affect other tests:

```suggestion
	common.RelaxPrecompileRangeForTest(false)
	enableTestExecutionSpecState(false)
	defer common.RelaxPrecompileRangeForTest(true)
	defer enableTestExecutionSpecState(true)
```